src/components/Plugin/SafeSnap/Modal/OptionApproval.vue

```diff
@@ -50,8 +56,17 @@
 
 <script>
 import { BigNumber } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
```

Comment:
Doing change just to trigger deployment
```suggestion
import { formatUnits } from '@ethersproject/units';

```